assets/src/edit-story/components/keyboardShortcutsMenu/getKeyboardShortcuts.js

```diff
@@ -126,9 +126,31 @@ function getKeyboardShortcuts() {
               { label: __('+ click', 'web-stories') },
             ],
           },
+          {
+            label: __('Move layer forward or backward', 'web-stories'),
```

Comment:
I wonder if the word `layer` is even necessary for this context -- since it has "Element" as the group title. Might be confusing, especially with Voice-Over. WDYT?

Reply:
It is to specify, that it's not actually moving the element itself, but the layer the element is in. I'm open to other copy. @o-fernandez, what do you think?

Reply:
I think that both: 
**Move forward or backward**
**Move to front or back**

Work well, so we can drop layer.